sled-diagnostics/src/queries.rs

```diff
@@ -22,11 +22,14 @@ use crate::contract_stub::ContractError;
 
 const DLADM: &str = "/usr/sbin/dladm";
 const IPADM: &str = "/usr/sbin/ipadm";
+const NVMEADM: &str = "/usr/bin/nvmeadm";
```

Comment:
This should be in sbin.

While testing this in a4x2 I noticed we were looking for nvmeadm in the wrong spot:
```
root@g0:~# curl http://[fd00:1122:3344:101::1]:12345/support/nvmeadm-info
{"success":{"command":"/usr/bin/pfexec /usr/bin/nvmeadm list","stdio":"pfexec: unable to execute /usr/bin/nvmeadm: No such file or directory\n","exit_status":"exit status: 1","exit_code":1}}
```

Reply:
Gah, thanks for catching that.